apps/masterbots.ai/components/layout/sidebar/sidebar.tsx

```
@@ -5,10 +5,25 @@ import { SidebarHeader } from '@/components/layout/sidebar/sidebar-header'
 import React from 'react'
+import { usePathname } from 'next/navigation'
+import { useThread } from '@/lib/hooks/use-thread'
 
 export function Sidebar({ className }: React.ComponentProps<'div'>) {
   const { isSidebarOpen, isLoading } = useSidebar()
+  const prevPathRef = React.useRef(usePathname());
+  const pathname = usePathname();
+  const {  setActiveThread, setIsOpenPopup } = useThread()
+  const rootAndChatRegex = /^\/(?:c)?$/;
 
+  React.useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: setActiveThread

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)